simple_history/models.py

```diff
@@ -655,6 +755,35 @@ def diff_against(self, old_history, excluded_fields=None, included_fields=None):
                 changes.append(ModelChange(field, old_value, current_value))
                 changed_fields.append(field)
 
+        # Separately compare m2m fields:
+        for field in old_history._history_m2m_fields:
+            # First retrieve a single item to get the field names from:
+            reference_history_m2m_item = (
+                getattr(old_history, field.name).first()
+                or getattr(self, field.name).first()
+            )
+            history_field_names = []
+            if reference_history_m2m_item:
+                # Create a list of field names to compare against.
+                # The list is generated without the primary key of the intermediate
+                # table, the foreign key to the history record, and the actual 'history'
+                # field, to avoid false positives while diffing.
+                history_field_names = [
+                    f.name
+                    for f in reference_history_m2m_item._meta.fields
+                    if f.name not in ["id", "m2m_history_id", "history"]
```

Comment:
Would it be more sensible to apply the same logic here as in #923 ?

Reply:
If you mean checking for the editable flag, yes, I think that makes sense.

https://github.com/jazzband/django-simple-history/pull/923/files#diff-a13c48090526a77430e6819c61ffb009b8ed15ffca1b40f3eef675c8c9b77db9R640

Reply:
I've added that and rebased on master!